tests/cl-genesis-positions/convert.go

```diff
@@ -3,7 +3,6 @@ package main
 import (
```

Comment:
All changes to files in the cl-genesis-positions folder are drive by changes to address lint issues that made it into main due to this folder not getting checked in CI